File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.py
def solution(s):
    answer = False
    s= list(s)
    flag = 0
    lenCheck = True if len(s) == 4 or len(s) == 6 else False
    for i in s:
        if i.isalpha():
            flag = 1
    if flag ==0 and lenCheck == True: 
        answer = True
    
    return answer

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        char[] chars = s.toCharArray();
        for (int i =0; i<s.length();i++){
            if ((chars[i] >= 'a' && chars[i] <= 'z') || (chars[i] >= 'A' && chars[i] <= 'Z'))
                return false;
        }
        if (s.length() == 4 || s.length() ==6)
            return true;
        return false;
    }
}